src/components/WorldListPage_WorldsForEnsOwnersFeature/WorldListPage.tsx

```diff
@@ -355,6 +367,12 @@ const WorldListPage: React.FC<Props> = props => {
     setPage(1)
   }, [tab])
 
+  useEffect(() => {
+    if (canOpenWorldsForENSOwnersAnnouncementModal()) {
+      onOpenWorldsForENSOwnersAnnouncementModal()
+    }
+  }, [onOpenWorldsForENSOwnersAnnouncementModal])
```

Comment:
If you want to make it simpler, you don't need a modal to be opened through the state using an action, you can simple instantiate the modal and render it if it satisfies the conditions. There are examples of this in the [Marketplace's FTU modals](https://github.com/decentraland/marketplace/blob/master/webapp/src/components/Modals/FTU/EmotesV2LaunchModal/EmotesV2LaunchModal.tsx).

Reply:
You are right, I just copied the implementation from the `EmotesV2AnnouncementModal`